app/components/Header.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import styles from './Header.module.css';
+import {Link} from 'react-router';
+
+function Header() {
+    return <Link className={styles.header} to="/">AquaBrain</Link>;
```

Comment:
Schreibweise beachten, ich glaube es war AquaBRain, sicher bin ich mir aber gerade nicht.

Reply:
Ich habe es so geschrieben, wie auf dem Mockup in der User Story. Wenn ich es ändern soll, bitte klären, wie es richtig geschrieben wird.

Reply:
Dann wird es so richtig sein, denke ich. @Jonasstemme wie siehst du das?

Reply:
Lt. Protokoll aus einem der Jourfix sollte es so geschrieben sein:

<img width="483" alt="image" src="https://github.com/user-attachments/assets/015fbf16-c373-4118-9456-aecfec820e7b">